mlserver/parallel/pool.py

```diff
@@ -23,6 +24,35 @@
 InferencePoolHook = Callable[[Worker], Awaitable[None]]
 
 
+class WorkerRegistry:
```

Comment:
nit: should this be in `registry.py`?

Reply:
I'd say it's a different type of registry. The registry in `registry.py` keeps track of the different worker pools running, whereas this registry just keeps track of which models have each worker loaded.